### DISCUSSION TITLE
GLB Compression Support (DRACO/Meshoptimizer)

### DISCUSSION THREAD

**Discussion by DirectiveCreator** [mml-io/mml#77] [Ideas]
**What is your idea?**
The idea is to implement DRACO and or Mesh optimizer compression support for loading GLBs in the web experience/playground, resulting in significantly reduced GLB sizes. The addition of a loading bar could also enhance user perception of loading speed.

**Why is it important?**
This would improve loading times and user experience. 

**What do you propose?**
Utilize existing (Three.js) libraries to enable Draco and Mesh optimizer compression support. 
By integrating these, we can efficiently reduce the sizes of loaded GLBs, improving the web experience.

- https://threejs.org/docs/#examples/en/loaders/DRACOLoader
- https://github.com/zeux/meshoptimizer

**Comment by MarcusLongmuir** [comment]
This is now tracked in https://github.com/mml-io/mml/issues/84

**Comment by MarcusLongmuir** [comment]
This was completed in #136 🎉
